src/main/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleGroupSyncMonitor.scala

```diff
@@ -156,7 +157,8 @@ class GoogleGroupSyncMonitorActor(
           import DefaultJsonProtocol._
           import WorkbenchIdentityJsonSupport._
           import org.broadinstitute.dsde.workbench.sam.google.SamGoogleModelJsonSupport._
-          logger.info(s"synchronized google group ${report.toJson.compactPrint}")
+          logger.info(s"synchronized google group", StructuredArguments.raw("syncDetail", report.toJson.compactPrint))
```

Comment:
just fyi, if you use `io.chrisdavenport.log4cats.StructuredLogger`, then you can do something like 
[this](https://github.com/broadinstitute/workbench-libs/blob/develop/google2/src/main/scala/org/broadinstitute/dsde/workbench/google2/package.scala#L84), basically add any additional context as a `Map[String, String]`

Reply:
I played around with `StructuredLogger` a bit and what is in the commit seemed to be the most natural fit for this code since it is `Future`y and not `IO`y

Reply:
I also wanted more than `Map[String, String]`. If you look at the examples you can see there is more structure there.

Reply:
yea, my comment was more of a general fyi...not specifically for this line